FreeRTOS/FreeRTOS
# Remove unused files

Description
-----------
Removed a 16MByte flash image file that was checked in by mistake (several years ago).
Remove the copies of lwIP that are no longer reference from demo projects.


Test Steps
-----------
Not applicable.

Related Issue
-----------
Not applicable.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
